tools/mo/openvino/tools/mo/load/caffe/loader.py

```diff
@@ -42,6 +45,8 @@ def load(self, graph: Graph):
         graph.graph['original_shapes'] = original_shapes
         graph.graph['caffe_pb2'] = caffe_pb2
 
+        if argv.k is None:
+            argv.k = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'front', 'caffe', 'CustomLayersMapping.xml')
         custom_layers_map = custom_layers_mapping.load_layers_xml(argv.k)
```

Comment:
why is this changed?

Reply:
It was changed to remove the default value of "k" and "caffe_parser_path" from convert_model() definition.
If we leave it in definition, then for tools.ovc.convert_model() we would need to search location of legacy convert_model and set these legacy paths. 
So I changed default values for  "k" and "caffe_parser_path" to None and added setting the default paths in in places when they are actually used.